person.rb

```diff
@@ -0,0 +1,18 @@
+class Person
+  attr_accessor :id, name, age
+
+
+  def can_use_services?
+    return unless age == 18 && parent_permission == true
+
+    true
+  end
+
+  private
+
+  def of_age?
+    return false if age <= 18
+
+    true
+  end
```

Comment:
- [ ] there should be a constructor with 'name', 'age', and 'parent_permission' as parameter. 'name' and 'parent_permission' are optional and should have default values of "Unknown" and 'true'. please find an example about constructors in a previous lesson under this [link](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/class_and_object_for_irb.md#constructor-and-instance-vars)